src/redux/saga/jobseeker.ts

```diff
@@ -0,0 +1,50 @@
+import { put, takeLatest, select } from 'redux-saga/effects'
+import axios from 'axios'
+import { AppState } from '../types'
+
+import {
+  registerJobseekerSuccess,
+  registerJobseekerFail,
+  loginJobseekerSuccess,
+  loginJobseekerFail,
+} from '../../redux/actions/jobseeker'
+
+const credential = (state: AppState) => state.jobseeker.credential
+function* registerJobseekerSaga() {
+  try {
+    const credentialData = yield select(credential)
+    const req = yield axios.post('/jobseeker', {
+      info: {},
```

Comment:
Remove

Reply:
Why? Don't I need to make a post request here in saga?